sdk/communication/Azure.Communication.CallAutomation/src/Models/AnswerCallOptions.cs

```diff
@@ -32,9 +32,9 @@ public AnswerCallOptions(string incomingCallContext, Uri callbackUri)
         public Uri CallbackUri { get; }
 
         /// <summary>
-        /// The endpoint URL of the Azure Cognitive Services resource attached
```

Comment:
The name CallIntelligenceOptions is not finalized yet, I will change this once the latest is available in swagger.